src/engine/strat_engine/filesystem.rs

```diff
@@ -64,6 +90,32 @@ impl StratFilesystem {
         self.thin_dev.id()
     }
 
+    /// Return an extend size for the filesystem over the thin device.
+    /// TODO: returning the current size will double the space provisoned to
+    /// the filesystem.  We should determine if this is a reasonable value.
+    fn extend_size(&self, current_size: DataBlocks) -> DataBlocks {
+        DataBlocks(*current_size * 2)
```

Comment:
The above was just a suggestion to use the defined operators.

But I think the _correct_ return value is actually just ```current_size```.